mnist_demo.ipynb

```diff
@@ -415,33 +415,42 @@
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {
     "collapsed": true
    },
-   "outputs": [],
-   "source": []
+   "source": [
+    "# Additional Resources"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To learn more, check out [this highly-regarded online class](http://cs231n.github.io).\n",
+    "\n",
+    "Tensorflow has a neat [demo playground](http://playground.tensorflow.org). "
+   ]
   }
  ],
  "metadata": {
   "anaconda-cloud": {},
   "kernelspec": {
-   "display_name": "Python [default]",
+   "display_name": "Python 2",
    "language": "python",
-   "name": "python3"
+   "name": "python2"
   },
   "language_info": {
    "codemirror_mode": {
     "name": "ipython",
-    "version": 3
+    "version": 2
```

Comment:
Instead of changing the python version I can add an import to convert all the prints

Python 3 is the future, python 2 has been marked for end of life by 2020